http/pact/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>pact</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: HTTP: Pact</name>
```

Comment:
I don't see this new module should be under the HTTP submodule? Maybe, testing-tooling/pact?

Reply:
If I understand correctly, Pact is used to describe a contract for servers and clients communicating over HTTP, that's why I put it into this submodule.
Also, as far as I can see, we don't have testing-tooling module. If you intended to create it, then I suggest we use something like third-party, since testing is not the only thing Patc can be used for and it is not a part of Quarkus. What do you think?

Reply:
You're not creating any HTTP endpoint. Instead, you're covering an integration with PACT tooling and the Quarkus tests. If not `testing-tooling`, maybe `contract-first/pact` suits better.